functions/openhab.bash

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env bash
 
+# two overlapping methods to enforce proper load order:
+# /var/lib/openhab2/etc/scripts/start-level.script to provide default start levels
```

Comment:
It seems a bit strange that this is _not_ done when you enable delayed rules, but only when you install a new openhabian version.
So we don't have is in existing installations, even if user selects  and also not in first_boot.
Maybe add the copy of start-level script also here if it does not exist...

Reply:
yep. I moved it to delayed_rules().
Now if we wanted to also enable it on existing installs that would require another call but my intention was to have it for new installs only.